src/tabbar.cpp

```diff
@@ -127,6 +129,32 @@ void TabBar::setNewTabButton()
     setTabButton(idx, QTabBar::RightSide, Q_NULLPTR);
 }
 
+int TabBar::realTabCount()
+{
+    // The last tab is "+" in TabBar, but that isn't a real tab which displays any content hence the real count is tab count - 1
+    if (count() - 1 < 0)
+        return 0;
+    return count() - 1;
+}
+
+void TabBar::moveToNextTab()
+{
+    auto index = currentIndex() + 1;
+    if (index == realTabCount()) {
+        index = 0;
+    }
+    setCurrentIndex(index);
+}
+
+void TabBar::moveToPreviousTab()
+{
+    auto index = currentIndex() - 1;
```

Comment:
You didn't address https://github.com/kiwix/kiwix-desktop/pull/867#discussion_r939554592

Reply:
Done now (though I had a hard time understanding what to do, let me know please if there's a better way!)